app/src/main/res/layout/more_details_fragment.xml

```diff
@@ -15,6 +15,7 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
+        android:startOffset="2000"
```

Comment:
it is only used when you have multiple animations on ObjectAnimator, animator or anim xmls, it helps you to await one animation for the amount of startOffset selected.